### DISCUSSION TITLE
我的理解是在 Notion / 飞书云文档 / ... / 本地 Markdown 文件夹 之间同步文章，对吗？

### DISCUSSION THREAD

**Discussion by bowencool** [LetTTGACO/elog#54] [Q&A] [answered]
如果是的话，那么是双向同步吗？

**Comment by LetTTGACO** [comment, marked as answer]
以前考虑过双向，但目前还是打算在 Elog 中做成单向的，原因有以下几点：
1. Elog 主要追求的可持续的自动化。基于大部分在线平台的写作体验都比本地/博客平台的好，Elog 是方便用户在写作平台到博客部署这中间的流程，遵循的是输入到输出的正向流程，也是为了提供更好的写作体验，所以在线写作平台 => 本地 markdown/博客站点是可持续的。
2. 双向同步也有一部分的需求，但不多，目前大部分都是为了从 A 写作平台迁移到 B写作平台之类需求，基本上是一次性的工作量。
3. 大部分的写作平台都支持 markdown 导入，即使用户想换，也可以利用 Elog 将其导出为符合格式的 md 文档，手动上传到新的写作平台。
4. 相较于从在线写作平台下载 md，上传文档到写作平台的复杂度更高，流程更复杂，目前没这么多精力搞这个。
5. Elog 未来的版本会进行模块拆分，旨在建立起生态，而在线写作平台之间的同步可能也会以插件的形式存在，今年的目标也会放在这上面，欢迎持续关注，更欢迎参与建设
